ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.18.yaml

```diff
@@ -113,6 +113,27 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-upi
+- as: azure-ipi-confidentialvm-vmgueststateonly-compute-f28
+  cron: 15 21 17 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_DC4ads_v5
+      ENABLE_CONFIDENTIAL_COMPUTE: "true"
```

Comment:
The setting only for COMPUTE maybe have different behaviors like `azure-ipi-confidentialvm-vmgueststateonly-f5` ?

Reply:
yes, Gen2 image definition set security type as `ConfidentialVMSupported` or `TrustedLaunchedSupported` (Azure recommended) in bug OCPBUGS-36670, so the image can be used to create either Gen2 VMs or confidential VMs.

`azure-ipi-confidentialvm-vmgueststateonly-compute` is to cover this scenario, only worker nodes are confidential VMs.